v2/cmd/asoctl/internal/importing/find_resources_by_scope.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package importing
+
+import (
+	"sync"
+
+	"github.com/Azure/azure-service-operator/v2/api"
+	"github.com/Azure/azure-service-operator/v2/internal/set"
+	"github.com/Azure/azure-service-operator/v2/pkg/genruntime"
+)
+
+var (
+	resourceTypesByScope         map[genruntime.ResourceScope]set.Set[string]
```

Comment:
minor, leave a comment explaining the set of `strings` is of `typeName`'s?

Reply:
Comment added.